FluffyDisplay/AppDelegate.swift

```diff
@@ -56,6 +56,7 @@ class AppDelegate: NSObject, NSApplicationDelegate, NetServiceDelegate, NetServi
       Resolution(1600, 1200, 125, false, "UXGA"),
       Resolution(1920, 1080, 102, false, "HD, 21.5-inch iMac"),
       Resolution(1680, 1050, 99,  false, "WSXGA+, Apple Cinema Display (20-inch), 20-inch iMac"),
+      Resolution(1600, 1024, 86,  false, "WSXGA, Apple Cinema Display (22-inch)"),
```

Comment:
Or for consistency with the other items should these be:
```suggestion
      Resolution(1680, 1050, 99,  false, "WSXGA+, 20-inch Apple Cinema Display, 20-inch iMac"),
      Resolution(1600, 1024, 86,  false, "WSXGA, 22-inch Apple Cinema Display"),
```